docs/plugins.rst

```diff
@@ -676,3 +676,34 @@ The Vega plugin uses the `Vega library <http://trifacta.github.io/vega>`_ to ren
 JSON objects directly in the Girder application. To use it, simply upload the JSON file as an item,
 and then set a "vega: true" metadata field on the item. The visualization will then be rendered
 directly in the item view.
+
+Server FUSE
+----------- 
+
+When this plugin is enabled *and* the appropriate configuration option is set,
+it mounts a user-space filesystem that allows reading any file in Girder as if
```

Comment:
maybe "mounts a read-only user-space filesystem" ?  I think its important to use the word 'read-only'